subprojects/core/src/integTest/groovy/org/gradle/internal/operations/logging/LoggingBuildOperationProgressIntegTest.groovy

```diff
@@ -386,9 +386,7 @@ class LoggingBuildOperationProgressIntegTest extends AbstractIntegrationSpec {
             output.addAll(it.progress(LogEventBuildOperationProgressDetails))
         }
 
-        def uniqueMessages = output.collect { it.details.message }.unique()
-        uniqueMessages.contains "started operation"
-        uniqueMessages.contains "finished operation"
+        output.collect { it.details.message }.unique() == []
```

Comment:
🤔 And the messages go away here, since they are emitted on the `listener started` message? What else do we call in these started listeners? Can somebody emit some logging there via a public API?